webcountriesapp/src/main/java/com/lambdaschool/webcountries/NameController.java

```diff
@@ -0,0 +1,63 @@
+package com.lambdaschool.webcountries;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+
+
+@RestController
+@RequestMapping("/names")
+public class NameController
+{
+    ///localhost:7777/names/all
+    @RequestMapping(value="/all",
+            produces={"application/json"})
+    public ResponseEntity<?> getAllCountries()
```

Comment:
For `GET` methods, **use `@GetMapping` instead of `@RequestMapping`** - `@RequestMapping` will allow other requests (like `DELETE`, for example) to run on your endpoint methods (as opposed to how `@GetMapping` is solely for `GET` requests)